src/ophyd_async/epics/areadetector/drivers/kinetix_driver.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from ..utils import ad_rw
+from .ad_base import ADBase
+
+
+class KinetixTriggerMode(str, Enum):
+    internal = "Internal"
+    edge = "Rising Edge"
+    gate = "Exp. Gate"
+
+
+class KinetixReadoutMode(str, Enum):
+    sensitivity = 1
+    speed = 2
+    dynamic_range = 3
+
+
+class KinetixDriver(ADBase):
+    def __init__(self, prefix: str) -> None:
+        super().__init__(prefix)
+        # self.pixel_format = ad_rw(PixelFormat, prefix + "PixelFormat")
+        self.trigger_mode = ad_rw(KinetixTriggerMode, prefix + "TriggerMode")
+        self.mode = ad_rw(KinetixReadoutMode, prefix + "ReadoutPortIdx")
```

Comment:
For areaDetector I've tended to name the attribute after the PV suffix, is there a reason to name this `mode` rather than `readout_port_idx` or `readout_mode`?

Reply:
I had `ReadoutMode` originally, but changed it because I couldn't make the linter happy actually...

I didn't want to use `ReadoutModeIdx` because I think treating it as a `ReadoutMode` selection rather than an index in the list is better.

Reply:
Could you point me to the corresponding record in the template please?

Reply:
https://github.com/NSLS-II/ADKinetix/blob/ee1359dbb02d99e74bd442e228a760f4f9e069b7/kinetixApp/Db/ADKinetix.template#L116

Reply:
Could these enum strings be pushed down to the template? Then having a `ReadoutMode` PV would make more sense...

I've tried not to add any translation in the "bucket of PVs" layer, and put the logic in the `Controller`, but I guess that is difficult here as the attribute isn't used in the controller. 

Either way, I think `readout_mode` for the attribute and `KinetixReadoutMode` are reasonable names, is the linter happy with those?